lib/bap_llvm/llvm_loader_stubs.c

```diff
@@ -0,0 +1,37 @@
+#include <caml/memory.h>
+#include <caml/fail.h>
+#include <caml/alloc.h>
+#include <caml/custom.h>
+#include <caml/bigarray.h>
+#include <caml/compatibility.h>
+#include <caml/callback.h>
+
+#include "llvm_loader.h"
+#include "llvm_loader_stubs.h"
+
+void failn(int n) {
```

Comment:
use `static` to prevent namespace pollution. All C functions that are not really exported (i.e., that are not part of the interface, should be static).